src/server/images/models.js

```diff
@@ -0,0 +1,26 @@
+import db from '../database';
+
+export default class ImageManager {
+  static fetchPrefix(prefix, cb) {
```

Comment:
static make to use method without creating an instance.
I just want to use this function without create instance That is it!